docs/buildah-add.1.md

```diff
@@ -89,7 +89,7 @@ buildah add containerID 'passwd' 'certs.d' /etc
 
 ## FILES
 
-### .containerignore
+### .containerignore/.dockerignore
```

Comment:
This appears to be a path on quick look maybe make it `.containerignore or .dockerignore`. Anyways is there any need to include `.dockerignore` in this heading ? I think the text below describes talks about `.dockerignore`.